package.json

```diff
@@ -2,10 +2,12 @@
   "name": "d3-react-sparkline",
   "version": "1.0.3",
   "description": "A simple component to render a sparkline with D3",
-  "main": "src/d3-react-sparkline.js",
+  "main": "dist/d3-react-sparkline.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node server.js",
+    "compile": "babel -d dist/ src/",
+    "prepublish": "npm run compile"
```

Comment:
This is the magic that means babel will run before a publish. Taken from here http://mammal.io/articles/using-es6-today/

It's a nice approach that allows those not using Babel / ES6 to use this project.